practices/backlog_cleanup.md

```diff
@@ -20,32 +20,35 @@ author_github_username: cadamini
 
 **TL;DR**
 
-A well-managed backlog results in a better overview of your important open issues and features, faster planning, and a cleaner scope. Periodically remove obsolete items from the backlog that have been created throughout the project lifecycle.
+Backlog Cleanup is important to keep a well-managed backlog. An ordered and short list of tasks, issues, and ideas results in a better overview of what's important, faster planning, and a cleaner scope. Periodically remove obsolete items from the backlog that have been created throughout the project lifecycle.
 
 ## What Is Backlog Cleanup  
 
-Backlog Cleanup describes the process of managing issues or ideas for new features in your product backlog. After some strategy meetings in the last months/quarters, some of the items in your backlog may be out of scope, since the product strategy has shifted in a different direction.
+Backlog Cleanup describes the process of managing tasks, issues, and ideas in your product backlog. You can reorder, rewrite, and delete backlog items. 
 
 ![Product backlog](/files/backlog-cleanup.jpg)
 [Close-up sticky note with product backlog](https://www.freepik.com/free-photo/close-up-sticky-note-with-product-backlog_7816260.htm#query=backlog&position=0&from_view=search)
 
 ## Why You Might Want Backlog Cleanup
 
-Both the prioritiation and the size of a product backlog is key. Your team will most probably work on the top items in the list. 
-When backlogs grow, it will be harder to see the relationship between items. Backlog cleanup will help you to keep track.
+Typically, companies have a project board, such as Trello, Jira, or GitHub projects. The boards contain columns for the different stages and the inbox or backlog column contains many different items. After some strategy meetings in the past months, some of the items in your backlog may be out of scope, since the product strategy or your team's priorities have shifted in a different direction.
```

Comment:
```suggestion
Typically, companies have a project board, such as GitHub projects, that contains columns for different stages. The inbox or backlog column contains many different items. After some strategy meetings in the past months, some of the items in your backlog may be out of scope, since the product strategy or your team's priorities have shifted in a different direction.
```